elixir/apps/web/lib/web/live/settings/identity_providers/okta/components.ex

```diff
@@ -83,14 +83,14 @@ defmodule Web.Settings.IdentityProviders.Okta.Components do
 
                 <div>
                   <.input
-                    label="OAuth Authorization Server URI"
+                    label="Okta Account Domain"
                     autocomplete="off"
-                    field={adapter_config_form[:oauth_uri]}
-                    placeholder="https://<company>.okta.com/.well-known/oauth-authorization-server"
+                    field={adapter_config_form[:okta_account_domain]}
+                    placeholder="<company>.okta.com"
```

Comment:
Just a minor thing, but what do you think of simplifying this to require them to only put in their `<company>` or tenant ID here, and we append `okta.com` ourselves? It might help prevent typo mistakes or (maybe in rare cases) typo squatting attacks for domains like `otka.com`

Reply:
Nevermind, I see now why you did it like this:

<img width="1506" alt="14-find-okta-account-domain" src="https://github.com/firezone/firezone/assets/167144/22b3d273-dfbd-4e09-b0d5-b64ce031c7eb">

Reply:
Yeah, having easy access to that drop down and copy button was the primary reason, but asking for the whole domain also allows customers to use their preview/sandbox domain or EMEA domain if they would ever want/need to. 

https://developer.okta.com/docs/concepts/okta-organizations/#org-urls